ansible/roles/marathon-server/tasks/main.yml

```diff
@@ -53,6 +53,32 @@
     tags:
       - packages
       - marathon
+ 
+  - name: Download auth plugin
+    get_url: url={{marathon_auth_plugin_url}} dest={{tmp_dir}}/{{marathon_auth_plugin_pkg}} sha256sum={{marathon_auth_plugin_pkg_sha256}}
+    tags:
+      - install
+      - plugin
+      - configuration
+
+  - name: Ensures {{base_data_dir}}/marathon/plugins dir exists
```

Comment:
Consider using `{{marathon_auth_plugin_location}} ` in the name also (instead of `{{base_data_dir}}/marathon/plugins`) so if it changes it is consistent.